tests/integration/test_runner_with_kafka.py

```diff
@@ -102,19 +102,30 @@ def test_write_read_to_kafka(start_service, subtests):
         try:
             runner.run()
         except ray.exceptions.RayActorError:
```

Comment:
```suggestion
        except ray.exceptions.RayActorError:
            # This is expected because we activated the poison pill
            pass
        ....
```
You should keep the same structure when doing this try-except block with a PoisonPill activation. (see comments in a previous PR: https://github.com/aineko-dev/aineko/pull/112#pullrequestreview-1794510059)

Reply:
Yes, that's more consistent. I updated.